### PR TITLE
Add DRY21 Modbus control script

### PR DESCRIPTION
**Pull request by anataty**

### CODE REVIEW

**Comment by nkrasko** [comment]
@anataty it worth to mention that DCN/IDC is required and that Modbus TCP of the Electrolyser is used.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
...
+
+Example scripts need pymodbus package to be installed. You can install it running: 
+`pip3 install pymodbus`
+
+
+
+## dry21_modbus_control.py
+#### Your devices (Enapter Electrolyser(s) EL 2.1 and Dryer DRY 2.1) must be connected to either [Dryer Control Network](https://handbook.enapter.com/electrolyser/el21a/el21a.html#dryer-control-network-with-enapter-cloud-connection) or [Isolated Dryer Control Network](https://handbook.enapter.com/electrolyser/el21a/el21a.html#isolated-dryer-control-network). 
+
+This script allows to control Enapter Dryer DRY 2.1 over Modbus TCP. Use it if you want Dryer to:
```
Comment (nkrasko) [inline README.md]: This script allows controlling Enapter Dryer  DRY 2.1 over Modbus TCP of Electrolyser unified with DCN/iDCN

**Review by dshil — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -1,2 +1,29 @@
-# dry21-scripts
-Enapter Dryer DRY 2.1 control scripts examples 
+## Introduction
+
+Enapter Dryer is a hybrid temperature/pressure swing adsorption system increasing hydrogen purity to 99,999%. Find out more about [Enapter Dryer](https://handbook.enapter.com/electrolyser/dry21/dry21.html).
+
+We developed [Dryer Control Network](https://handbook.enapter.com/electrolyser/el21a/el21a.html#%F0%9F%95%B8%EF%B8%8F-dryer-control-network) - technology that allows you to control and monitor Enapter Dryer from [Electrolyser](https://handbook.enapter.com/electrolyser/el21a/el21a.html) Web GUI or Modbus TCP interfaces. Also when Dryer Control Network is enabled, Dryer starts automatically when at least one of connected electrolysers is steady.
```
Comment (anataty) [inline README.md]: Maybe we change links to handbook using our [link shortener](https://go-admin.enapter.com)?

File: examples/dryer_control_over_modbus/control.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            else:
+                print("Reboot command is sent to Dryer DRY 2.1")
+                break
+        except exceptions.ConnectionException:
+            connection_troubleshooting()
+            exit(1)
+
+
+def read_once(client, register) -> None:
+    states = {
```
Comment (anataty) [inline examples/dryer_control_over_modbus/control.py:46]: @dshil this list is enough?

Comment (dshil) [reply examples/dryer_control_over_modbus/control.py:46]: This is a subset of [all states](https://handbook.enapter.com/electrolyser/dry21/dry21.html#logic-states). Here we use some of them, just for a demonstration purpose.

File: README.md (hunk 2 above)
Comment (dshil) [reply README.md]: Fixed.

**Comment by dshil** [comment]
@anataty ping.

**Pull request merged**